ni_measurementlink_service/session_management/_reservation.py

```diff
@@ -568,6 +568,7 @@ def __init__(
         multiplexer_session_info: Optional[
             Sequence[session_management_service_pb2.MultiplexerSessionInformation]
         ] = None,
+        pin_or_relay_group_mappings: Optional[Dict[str, Iterable[str]]] = None,
```

Comment:
FYI, `str` is an `Iterable[str]` that iterates over its characters. I don't know of a good way to accept `Iterable[str]` but exclude `str`, though.

Reply:
As per this open [issue](https://github.com/python/typing/issues/256) in Python, I don't believe there is any alternative to address this case. Additionally, we have been using `Iterable[str]` in other places in our Python repository as well. Therefore, it seems that `Iterable[str]` would be suitable. Please let me know if you have other thoughts.